Pod/Classes/Swift/VOKCoreDataManager+Swift.swift

```diff
@@ -27,15 +50,33 @@ public extension VOKCoreDataManager {
     }
     
     /**
+     DEPRECATED - use importArray(_:of:withContext:) instead.
      Deserializes an array of dictionaries full of strings and creates (or updates) instances of a managed object subclass in the given context.
      
      - parameter inputArray: An array of dictionaries defining managed object subclasses
      - parameter objectClass: The class of objects to create
      - parameter context: The managed object context in which to create the objects or nil for the main context (defaults to nil)
      - returns: A typed Array of created or updated objects
      */
+    @available(*, deprecated: 4.1.0, message: "use importArray(_:of:withContext:) instead")
     public func importArray<T: NSManagedObject>(_ inputArray: [[String : Any]],
-        forClass objectClass: T.Type,
+                            forClass objectClass: T.Type,
```

Comment:
(However I will abide by house rules - @brockboland if you want me to leave as is, will do. otherwise I can line everything up)